.packit.yaml

```diff
@@ -5,22 +5,32 @@ specfile_path: beaker.spec
 
 # add or remove files that should be synced
 synced_files:
-    - beaker.spec
-    - .packit.yaml
+  - beaker.spec
+  - .packit.yaml
 
 notifications:
   pull_request:
     successful_build: false
 
 jobs:
-- job: copr_build
-  trigger: pull_request
-  metadata:
-    targets:
-      - epel-8-x86_64
-      - fedora-33-x86_64
-      - fedora-34-x86_64
-      - fedora-35-x86_64
-      - fedora-rawhide-x86_64
-      - centos-stream-8-x86_64
-      - centos-stream-9-x86_64
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - epel-8-x86_64
+        - epel-9-x86_64
+        - centos-stream-8-x86_64
+        - centos-stream-9-x86_64
+
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - fedora-all-x86_64
+    actions:
+      post-upstream-clone:
```

Comment:
This shouldn't be needed. The spec should be working out of the box for Fedora targets, otherwise, we will struggle to build packages in Fedora dist-git again.

Reply:
it is what was needed to get packit green for fc39/rawhide, feel free to mess with it 😂

Reply:
Yeah, I completely understand your point. I created a separate issue for this 🚀